src/org/ods/services/OpenShiftService.groovy

```diff
@@ -137,8 +137,11 @@ class OpenShiftService {
         valuesFiles.collect { upgradeFlags << "-f ${it}".toString() }
         values.collect { k, v -> upgradeFlags << "--set ${k}=${v}".toString() }
         if (withDiff) {
-            def diffFlags = upgradeFlags.findAll { it != '--atomic' }
+            def diffFlags = upgradeFlags.findAll { it != '--atomic' || it != '--three-way-merge'}
```

Comment:
I do not understand this line. You wouldn't need to check since you handle the case that `--three-way-merge` is already given down below, no? Apart from that, `--three-way-merge` isn't a known flag for `helm upgrade`.

`--atomic` is filtered out because it is a known flag for `helm upgrade`, but it is not handled by `helm diff`. But as I said I think it's better to set the env var to ignore unknown flags in diff, otherwise you prevent most upgrade flags to be used as it would break helm diffing.